WcaOnRails/app/views/persons/_results_by_event.html.erb

```diff
@@ -42,7 +42,19 @@
                     <% end %>
                   </td>
                   <td class="round"><%= t "rounds.#{result.roundTypeId}.cellName" %></td>
-                  <td class="place"><%= result.pos %></td>
+                  <% if result.roundTypeId == 'f' || result.roundTypeId == 'c' %>
```

Comment:
Do we need to consider DNF results (both single and average are DNFs)?

Reply:
I see I have missed that case. 
So will this check be good enough? 
`<% if (result.roundTypeId == 'f' || result.roundTypeId == 'c') && 
                  result.best_solve.clock_format != "DNF" %>`